rllib/algorithms/simple_q/simple_q_torch_policy.py

```diff
@@ -87,13 +86,13 @@ def compute_actions(
         )
         # Return (exploration) actions, state_outs (empty list), and extra outs.
         return (
-            actions,
+            np.array(actions),
             [],
             {
-                "q_values": q_vals,
-                SampleBatch.ACTION_LOGP: logp,
-                SampleBatch.ACTION_PROB: torch.exp(logp),
-                SampleBatch.ACTION_DIST_INPUTS: q_vals,
+                "q_values": q_vals.detach().numpy(),
```

Comment:
The output of this function is directly used for the dummy loss initialization, which needs numpy arrays.